go.mod

```diff
@@ -1,12 +1,13 @@
 module github.com/rancher/steve
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.1
 
 replace (
 	github.com/crewjam/saml => github.com/rancher/saml v0.2.0
 	github.com/knative/pkg => github.com/rancher/pkg v0.0.0-20181214184433-b04c0947ad2f
 	github.com/matryer/moq => github.com/rancher/moq v0.0.0-20190404221404-ee5226d43009
-	k8s.io/client-go => github.com/rancher/client-go v1.29.3-rancher1
```

Comment:
Are you sure that we don't need the changes from the client-go fork in steve?

Reply:
was about to update the PR comment...it was discussed in PR comment : https://github.com/rancher/client-go/pull/11#issuecomment-2036238634
so with k8s 0.30 client go all fixes have gone into upstream.

Reply:
Thanks, I'll leave this thread open so that other reviewers can see it easily, but if all the fixes are upstream then that's a good change.

Reply:
https://github.com/rancher/security-team/issues/349#issue-1820307873 has more context on why our custom changes are no longer required.